src/components/MonthPickerNavigation.jsx

```diff
@@ -0,0 +1,79 @@
+import React, { PropTypes } from 'react';
```

Comment:
That's it. Your solution is perfect, now the "PickerNavigation" component can be reused in the DayPicker and the MonthPicker. I had to rename and modify the imports of the components and the CSS code. The tests has been adapted to this solution too.